projects/optic/src/commands/oas/captures/getInteractions.ts

```diff
@@ -8,12 +8,61 @@ import { HarEntries } from './streams/sources/har';
 import { trackWarning } from '../lib/sentry';
 import * as AT from '../lib/async-tools';
 import { PostmanCollectionEntries } from './streams/sources/postman';
+import { OpenAPIV3 } from '@useoptic/openapi-utilities';
+
+async function* handleServerPathPrefix(
```

Comment:
Moved this function up to the top level interactions handler - we now adjust the path based on the OpenAPI server key and do not emit interactions when not relevant. This was in multiple places in the code base and was inconsistently implemented (implemented in some places and not other)